packages/@uppy/locales/src/fi_FI.js

```diff
@@ -0,0 +1,152 @@
+const fi_FI = {}
+
+fi_FI.strings = {
+  addMore: 'Lisää',
+  addMoreFiles: 'Lisää tiedostoja',
+  addingMoreFiles: 'Lisätään tiedostoja',
+  allowAccessDescription: 'Jotta voit ottaa kuvia tai videota kamerallasi, sinun tulee antaa tälle sivustolle oikeus käyttää kameraasi.',
+  allowAccessTitle: 'Salli kameran käyttö, kiitos',
+  authenticateWith: 'Mene %{pluginName}',
+  authenticateWithTitle: '%{pluginName} vaadittu tunnistautumiseen, jotta voit valita tiedostoja',
+  back: 'Takaisin',
+  browse: 'selaa',
+  cancel: 'Peruuta',
+  cancelUpload: 'Peruuta lähetys',
+  chooseFiles: 'Valitse tiedostot',
+  closeModal: 'Sulje ikkuna',
+  companionAuthError: 'Käyttöoikeus vaadittu',
+  companionError: 'Yhdistäminen Companioniin epäonnistui',
+  complete: 'Valmis',
+  connectedToInternet: 'Yhdistetty Internettiin',
+  copyLink: 'Kopioi linkki',
+  copyLinkToClipboardFallback: 'Kopioi alla oleva linkki',
+  copyLinkToClipboardSuccess: 'Linkki kopioitu leikepöydälle',
+  creatingAssembly: 'Valmistellaan lähetystä...',
+  creatingAssemblyFailed: 'Transloadit: Assemblyn luonti epäonnistui',
+  dashboardTitle: 'Tiedoston Lataaja',
+  dashboardWindowTitle: 'Tiedoston latausikkuna (Paina Esc sulkeaksesi)',
+  dataUploadedOfTotal: '%{complete} / %{total}',
+  done: 'Valmis',
+  dropHereOr: 'Pudota tiedostot tähän tai %{browse}',
+  dropHint: 'Pudota tiedostot tähän',
+  dropPaste: 'Pudota tiedostot tähän, liitä tai %{browse}',
+  dropPasteImport: 'Pudota tiedostot tähän, liitä, %{browse} tai tuo',
+  edit: 'Muokkaa',
+  editFile: 'Muokkaa tiedostoa',
+  editing: 'Muokataan %{file}',
+  emptyFolderAdded: 'Ei lisätty tiedostoja tyhjästä kansiosta',
+  encoding: 'Koodataan...',
+  enterCorrectUrl: 'Epäkelpo osoita: Varmista, että osoite osoittaa suoraan tiedostoon',
+  enterUrlToImport: 'Anna osoite tuodaksesi tiedoston',
+  exceedsSize: 'Tiedoston koko ylittää sallitun maksimin',
+  failedToFetch: 'Companion ei voinut ladata tiedostoa osoitteesta, onko osoite varmasti oikea?',
+  failedToUpload: 'Ei voitu lähettää tiedostoa %{file}',
+  fileSource: 'Tiedoston lähde: %{name}',
+  filesUploadedOfTotal: {
+    '0': '%{complete} / %{smart_count} tiedostosta lähetetty',
+    '1': '%{complete} / %{smart_count} tiedostoa lähetetty',
+    '2': '%{complete} / %{smart_count} tiedostoa lähetetty'
+  },
+  filter: 'Suodata',
+  finishEditingFile: 'Lopeta tiedoston muokkaus',
+  folderAdded: {
+    '0': 'Lisätty %{smart_count} tiedosto kansiosta %{folder}',
+    '1': 'Lisätty %{smart_count} tiedostoa kansiosta %{folder}',
+    '2': 'Lisätty %{smart_count} tiedostoa kansiosta %{folder}'
+  },
+  import: 'Tuo',
+  importFrom: 'Tuo kohteesta %{name}',
+  link: 'Linkki',
+  loading: 'Ladataan...',
+  logOut: 'Kirjaudu ulos',
+  myDevice: 'Minun laite',
+  noFilesFound: 'Sinulla ei ole tiedostoja tai kansioita täällä',
```

Comment:
Fine to use “sinulla” vs “teillä” here, doesn’t sound rude?

Reply:
"Sinulla" doesn't sound rude at all. But finnish may be sometimes very difficult to translate from english. I will check how finnish sentences work in different contexts after deploying uppy in my project.

Reply:
Ok, thank you! Merging, please feel free to fix something afterwards.